cmd/pipecd/ops.go

```diff
@@ -34,6 +36,7 @@ type ops struct {
 	httpPort    int
 	adminPort   int
 	gracePeriod time.Duration
+	enable      bool
```

Comment:
nit: `enableInsightCollector`

Reply:
Ah, now I have understood [this comment](https://github.com/pipe-cd/pipe/pull/1287#discussion_r547128299).